bundles/org.openhab.binding.ws980wifi/src/main/java/org/openhab/binding/ws980wifi/internal/discovery/ws980wifi.java

```diff
@@ -0,0 +1,284 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ws980wifi.internal.discovery;
+
+import static java.lang.Math.toIntExact;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.SocketTimeoutException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Joerg Dokupil - Initial contribution
+ */
+@NonNullByDefault
+public class ws980wifi {
```

Comment:
Class names should start with an uppercase character.

Reply:
you are right. I have started to modify the name to WS980WIFI, but it makes many modifications neccessary.
I want to leave the class name as it is.

Reply:
In eclipse you can rename this by placing your cursor on the name and `ctrl-shift-r` (or right-click mouse for menu.) This rename will change the filename, and all references where the class is used.

Reply:
Rename is ``alt-shift-r`

Reply:
you are right, but I want to stay with the naming, because the change effort ist to big in comparison with the readability improvement.